sdv/sampler.py

```diff
@@ -33,11 +34,25 @@ def reset_indices_tables(sampled_tables):
 
         return sampled_tables
 
-    def transform_synthesized_rows(self, synthesized_rows, table_name, num_rows):
+    @staticmethod
+    def _rescale_values(column):
+        """Scale an array of values in the interval [0, 1].
+
+        Args:
+            column (pandas.Series): Column values to scale
+
+        Return:
+            pandas.Series: Column values scaled into [0,1]
+        """
+        scaler = MinMaxScaler()
+        scaled = scaler.fit_transform(column.values.reshape(-1, 1)).ravel()
```

Comment:
Since this scaling will be different at every call depending on the sampled values, aren't we altering probability distributions here?

Reply:
Yes, right. I changed the implementation.